### ISSUE TITLE
Suggestion: option to change the telegram message destination

### ISSUE THREAD

**Issue by felpsio** [marckohlbrugge/wip-menubar#6]
Many tasks I complete I prefer to send privately to @wipbot instead of sending to the WIP group. Sometime I think it's hard to follow all the messages from the group because of the big number of new messages that keeps arriving.

So I would like to suggest to add a preference on the menu to choose the default destination: private wipbot or the wip group chat

**Comment by marckohlbrugge** [comment]
Good idea! Will add.
